templates/bamboo/0/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+bamboo-data:
+    image: busybox
```

Comment:
@matisku Can you add `net:none`, also you may want to run this container only once